src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+# Frame V4 API FAQ
+
+The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work for every step of the asset delivery lifecycle from plan to deliver.
+
+
+* How do I **authenticate**?
+
+    The first step in accessing the [Frame.io](http://frame.io/) API is getting authenticated. Create the user authentication credentials and then the access token. Adobe Identity Management Service (IMS) is Adobe’s OAuth 2.0-based identity management feature that supports authentication. There are two steps required for generating the access token:
+
+    * The user has to authorize the application before the token generation. The details for this can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#authorize-request).
+    * The next step is fetching the access token. The above step returns a **code** in the response body. This is used in this step. The exact endpoints to trigger and the parameters to use can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#fetching-access-tokens).
+* Where is the **API reference**?
+    * Go [here](https://developer-stage.adobe.com/frameio/api/current/) to see the Frame V4 API reference pages. 
+* How can I get **help**?
+    * Reach out to the [Frame.io](http://frame.io/) support team for assistance by chatting with us on our [Support Site](https://help.frame.io/en/). Please note that enterprise customers may be directed to their Customer Success Manager.
```

Comment:
```suggestion
    * Reach out to the Frame.io support team for assistance by chatting with us on our [Support Site](https://help.frame.io/en/). Please note that enterprise customers may be directed to their Customer Success Manager.
```
I guess Quip helpfully converted these into links for you when you exported as markdown, but we should probably fix all of these before merging / publishing.

Reply:
Done!